func.go

```diff
@@ -103,17 +136,72 @@ func fileLastModified(file string) time.Time {
 func fileGetBytes(file string) []byte {
 	data, err := ioutil.ReadFile(file)
 	if err != nil {
-		// Swallow errors inside of a template.
-		return nil
+		if Strict {
+			panic(err.Error())
+		}
+		return []byte{}
 	}
 	return data
 }
 
 func fileGetString(file string) string {
 	data, err := ioutil.ReadFile(file)
 	if err != nil {
-		// Swallow errors inside of a template.
+		if Strict {
+			panic(err.Error())
+		}
 		return ""
 	}
 	return string(data)
 }
+
+type relativeInclude func(include string) string
+
+func include(delims []string, callingFile string, ctx interface{}) relativeInclude {
```

Comment:
I think, `delims` and `ctx` can be accessed from public variables. They are read-only in all templates.